sound/components/sound_components.mk

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright 2024, UNSW
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+# This Makefile snippet builds the sound RX and TX virtualisers
+# it should be included into your project Makefile
+#
+# NOTES:
+#  Generates sound_virt.elf
+#
+
+
+SND_IMAGES := sound_virt.elf
```

Comment:
Is this used anywhere?

Reply:
Offline: yes, it is used on line 36
https://github.com/au-ts/sddf/pull/154/files#diff-02cc5ea9faacab31e55bdece3fba8034233f5378fe1b6c18f5903483ca734d00R36